tests/data/test_file.txt.md5

```diff
@@ -0,0 +1 @@
+169293f7c9138e4b50ebcab4358dc509
```

Comment:
I tried to use sha1, but midas responded with "The bitstream does not exist or you do not have the permission".  I didn't see anywhere obvious what hashing functions midas supports.